tests/unit/tasks/test_python_task.py

```diff
@@ -0,0 +1,39 @@
+from griptape.tasks import PythonTask
+from griptape.tasks.python_task import hello_world, deliberate_exception
```

Comment:
Per earlier comment, we can move these functions in here and out of the main class definition. Then you can eliminate the import.